packages/shared/lib/core/i18n/i18n.ts

```diff
@@ -0,0 +1,94 @@
+import { get } from 'svelte/store'
+import { addMessages, dictionary, getLocaleFromNavigator, init, _, getDateFormatter } from 'svelte-i18n'
+
+import { appSettings } from 'shared/lib/appSettings'
+
+import { DEFAULT_LOCALE_OPTIONS, SUPPORTED_LOCALES } from './constants'
+import { LocaleOptions } from './types'
+
+/*
+ * Code following https://phrase.com/blog/posts/a-step-by-step-guide-to-svelte-localization-with-svelte-i18n-v3/
+ */
+
+function supported(locale) {
```

Comment:
It would be nice to have parameter and return types in all of the functions in these files